src/styles/BaseStyles/src/tokens-button.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  --denhaag-button-border-radius: 3px;
+  --denhaag-button-border-width: 0.0625rem;
+  --denhaag-button-focus-border-color: var(--denhaag-ocher-4);
```

Comment:
Using `var(--denhaag-ocher-4)` and the other brand design tokens in this file seems fine to me.